handsontable/src/core.js

```diff
@@ -2055,14 +2062,21 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   };
 
   /**
-   * Loads new data to Handsontable. Loading new data resets the cell meta.
-   * Since 8.0.0 loading new data also resets states corresponding to rows and columns
-   * (for example, row/column sequence, column width, row height, frozen columns etc.).
+   * The `loadData()` method:
+   * - Replaces Handsontable's [`data`](@/api/options.md#data)
+   * - Resets cells' [configuration options](@/guides/getting-started/setting-options.md)
+   * - Resets rows' and columns' states (e.g. [row height](@/guides/rows/row-height.md), [column width](@/guides/columns/column-width.md), frozen [rows](@/guides/rows/row-freezing.md) and [columns](@/guides/columns/column-freezing.md) etc.)
```

Comment:
Same as the comments above.

Reply:
@jansiegel - please have a look now at the updated version: 64e6f14